index.php

```diff
@@ -1,5 +1,6 @@
 <?php
 require_once './vendor/autoload.php';
+require_once './src/Suppliers.php';

```

Comment:
I see you still have supplies as a class, please turn it to  a file that contains an array  of URLs